cmd/action/provider.go

```diff
@@ -75,3 +80,63 @@ func sqliteProvider(dsn string) (*Driver, error) {
 		Closer:      db,
 	}, nil
 }
+
+var reDockerConfig = regexp.MustCompile("(mysql|mariadb)(?::([0-9a-zA-Z.-]+)?(\\?.*)?)?")
```

Comment:
Can we be URL compliant and use`url.Parse`?

Reply:
I'll check